packages/paste-core/components/button-group/src/ButtonGroup.tsx

```diff
@@ -65,11 +63,19 @@ export interface ButtonGroupProps extends HTMLPasteProps<"div"> {
    * @memberof ButtonGroupProps
    */
   attached?: boolean;
+  /**
+   * Sets the `justify-content` CSS property. Only used for unattached button groups.
+   *
+   * @default "flex-start"
+   * @type {JustifyOptions}
+   * @memberof ButtonGroupProps
+   */
+  justifyContent?: JustifyOptions;
```

Comment:
question, do we need to expose this prop? For the justify storybook would we really want users to be able to use the display between. We could just update the default value and use the attached boolean to determine whether is need sot be different for the 2 supported variants

Reply:
hmm, you mean making unattached always centered and attached always flex-start? that would be a breaking design change. maybe I'm misunderstanding

Reply:
I mean leave the styling to how it currently is without any change when attached an then only apply the new flex when they are unattached. No new props for consumers